src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/extension/AbstractNameValueHeader/rowHeader.jelly

```diff
@@ -8,7 +8,7 @@
 
     <div class="pipeline-info">
         <div class="revision rounded">
-            <div class="title">Pipeline ${buildGrid.get(0,0).getRevision()}</div>
+            <div class="title build-parameters-trigger-id" data-build-id="${buildGrid.get(0,0).getId()}" id="build-parameters-trigger-${buildGrid.get(0,0).getId()}">Pipeline ${buildGrid.get(0,0).getRevision()}</div>
```

Comment:
I don't see the need for a new class here. Can select the existing class with high specificity using e.g. `.pipeline-info .revision .title`.